packages/server/src/fhir/sql.ts

```diff
@@ -150,7 +150,7 @@ export class Column {
 }
 
 export class Literal {
-  constructor(readonly value: string) {}
+  constructor(readonly value: any) {}
```

Comment:
I don't see any cases where this was necessary?

```suggestion
  constructor(readonly value: string) {}
```